### PR TITLE
Fix jwt rule note

### PR DESCRIPTION
**Pull request by shamsher31**
To fix https://github.com/istio/istio.io/issues/9914

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Comment by shamsher31** [comment]
cc: @nrjpoddar @linsun

File: security/v1beta1/jwt.proto [hunk 1]
```diff
@@ -89,13 +89,13 @@ message JWTRule {
   //
   // Example: `https://www.googleapis.com/oauth2/v1/certs`
   //
-  // Note: Only one of jwks_uri and jwks should be used. jwks_uri will be ignored if it does.
+  // Note: Only one of `jwks_uri` and `jwks` should be used.
```
Comment (hzxuzhonghu) [inline security/v1beta1/jwt.proto:92]: jwksUri

Comment (shamsher31) [reply security/v1beta1/jwt.proto:92]: I have seen `jwks_uri` mentioned in multiple release notes [this](https://github.com/istio/istio/pull/30014/files#diff-01aa61a7126fd7b057a98c3354466573817b5610e443038c0fcaab72887fa2d7R8) and [this](https://github.com/istio/istio/pull/30679/files#diff-9f7f93e2212215ca68c7d07ad7f19b0343ba73b6729c2f59d2b1887f6cf5186dR8)

Also `RequestAuthentication` CRD has both `jwks_uri` and `jwksUri`, so IMO one of them will be deprecated.

Any reason we should write `jwksUri` here?

Comment (howardjohn) [reply security/v1beta1/jwt.proto:92]: They are not two different fields, just two different representations of the same field. I believe `jwks_uri` is not valid in YAML so we probably don't want to use it in docs

Comment (hzxuzhonghu) [reply security/v1beta1/jwt.proto:92]: Yes, the proto is to developer, but the comments are to user

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**